samples/drivers/jesd216/boards/gd32a503v_eval.conf

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) 2022 YuLong Yao <feilongphone@gmail.com>
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+CONFIG_GPIO=y
```

Comment:
too late, but why doesn't board enable GPIO, as per-policy? this would not be needed at all.

Reply:
Sorry, I didn't know about this policy before.
Are all boards supposed to have the following peripherals enabled by default?

``` kconfig
CONFIG_CONSOLE=y
CONFIG_UART_CONSOLE=y
CONFIG_SERIAL=y

CONFIG_GPIO=y
```

Reply:
Yes, in general GPIO/UART are considered "minimal peripherals" used by lots of samples.